src/Generator.Rewrite/Program.cs

```diff
@@ -61,97 +60,91 @@ static void Main(string[] args)
 
         static bool dllimport;
 
-        void Rewrite(string file, string keyfile, IEnumerable<string> options)
+        void Rewrite(string file, string keyfile, List<string> options)
```

Comment:
If this signature was changed, we definitely want to change it to `IReadOnlyList<T>` or `IReadOnlyCollection<T>`. If neither is available, due to this running on ancient, decrepit versions of .Net, keep the original signature and perform the .ToList() inside the scope of the method to ensure that the enumration of the parameter only takes place once.